### DISCUSSION TITLE
I225-V无法识别

### DISCUSSION THREAD

**Discussion by zhangye151** [immortalwrt/immortalwrt#772] [Q&A]
I225-V无法识别

**Comment by ksong008** [comment]
@zhangye151 编译时候选上驱动

**Comment by 1715173329** [comment]
假设您用的是 21.02.1 正式版本，可以使用由 Nick 提供的固件： https://t.me/ctcgfw_openwrt_discuss/730396
您也可以手动下载 kmod-igc 驱动安装，或通过 imagebuilder 自行打包。
此外，在最新的 21.02 分支中，该驱动已默认集成。
